src/app/components/SearchBar.vue

```diff
@@ -38,17 +34,20 @@ export default {
   },
 
   computed: {
+    ...mapGetters(['nameOfItemsToSearch']),
```

Comment:
I have a suggestion to think about the simplification of such keys.
This a bit hard to read, and it's potential errors by typos
How about something like `searchType` or similar